src/docs/development/tools/devtools/memory.md

```diff
@@ -10,19 +10,28 @@ Allocated Dart objects created using a class constructor
 live in a portion of memory called the heap.
 
 DevTools Memory pane lets you peek at how an isolate is using
-memory at a given moment. This pane, using Snapshot and
+memory at a given moment. This pane using Monitor Allocations and
 Reset, can display accumulator counts. The accumulators
 can be used to study the rate of memory allocations,
 if you suspect your application is leaking memory or has
 other bugs relating to memory allocation.
 
-Memory profiling consists of four parts, each increasing
+Snapshot is a complete view of all Dart memory. Additionally, an analysis
+of the snapshot is also done.  The analysis looks for Flutter and Dart
+memory patterns that may cause leaks and could eventually crash a Flutter
+application.  For example, loading large images, megabyte size, when a
+thumbnail or something much smaller would have been much more efficient.
+
+Memory profiling consists of six parts, each increasing
 in granularity:
 
 * Memory overview chart
-* Event timeline
-* Snapshot classes
-* Class instances
+* Android specific memory chart (via Android's ADB tool)
+* Event timeline e.g., garbage collection (GC), and a user's memory tooling interactions
+* Monitoring of Dart total instances/bytes size totals and managing accumulators to detect memory leaks
+* Analysis of a Snapshot to detect possible Flutter memory problems 
```

Comment:
Snapshot analysis to..

remove Flutter

Reply:
Done.